src/boat_simulator/boat_simulator/nodes/physics_engine/physics_engine_node.py

```diff
@@ -318,6 +324,23 @@ def __publish(self):
         self.__publish_kinematics()
         self.__publish_counter += 1
 
+    def __update_sim_wind_sensor(self):
+        """Updates wind parameter of wind sensor"""
+        self.__sim_wind_sensor.wind = self.__wind_generator.next()
```

Comment:
I think it might be best to move this line to the beginning of `__update_boat_state()` and remove this method.